PWGGA/GammaConv/AliAnalysisTaskGammaCalo.cxx

```diff
@@ -3785,6 +3905,26 @@ void AliAnalysisTaskGammaCalo::CalculatePi0Candidates(){
         if((((AliConversionMesonCuts*)fMesonCutArray->At(fiCut))->MesonIsSelected(pi0cand,kTRUE,((AliConvEventCuts*)fEventCutArray->At(fiCut))->GetEtaShift(),gamma0->GetLeadingCellID(),gamma1->GetLeadingCellID()))){
           if(fLocalDebugFlag == 1) DebugMethodPrint1(pi0cand,gamma0,gamma1);
           fHistoMotherInvMassPt[fiCut]->Fill(pi0cand->M(),pi0cand->Pt(), tempPi0CandWeight);
+          if(fDoJetAnalysis && fConvJetReader->GetNJets()>0){
```

Comment:
if fConvJetReader does not exist, then it will crash at this point. Please make first a if(fDoJetAnalysis) and then inside of this condition another fConvJetRead->GetNJets()>0.